api/v1alpha1/validatorconfig_types.go

```diff
@@ -42,24 +45,21 @@ type Sink struct {
 
 // HelmRelease defines the configuration for a Helm chart release.
 type HelmRelease struct {
-	// Chart defines the Helm chart to be installed.
-	Chart HelmChart `json:"chart" yaml:"chart"`
+	// Name of the Helm chart.
+	Name string `json:"name" yaml:"name"`
+
```

Comment:
We need to track repository here in addition to the name:
```go
// Repository containing the Helm chart.
Repository string `json:"repository" yaml:"repository"`
```

Structure of Helm URI: `<registry>/<repository>/<chart>`